url-updater/src/main/java/com/devonfw/tools/ide/url/tool/kotlinc/KotlincUrlUpdater.java

```diff
@@ -34,4 +34,14 @@ protected Pattern getVersionPattern() {
 
     return Pattern.compile("[0-9]+\\.[0-9]+\\.[0-9]+");
   }
+
+  @Override
+  public String getCpeVendor() {
+    return "jetbrains";
+  }
+
+  @Override
+  public String getCpeProduct() {
+    return "kotlinc";
```

Comment:
See https://nvd.nist.gov/vuln/detail/CVE-2022-24329#vulnConfigurationsArea
```suggestion
    return "kotlin";
```